config/application.yml.default

```diff
@@ -277,6 +277,7 @@ development:
   password_pepper: f22d4b2cafac9066fe2f4416f5b7a32c
   piv_cac_verify_token_secret: ee7f20f44cdc2ba0c6830f70470d1d1d059e1279cdb58134db92b35947b1528ef5525ece5910cf4f2321ab989a618feea12ef95711dbc62b9601e8520a34ee12
   rails_mailer_previews_enabled: 'true'
+  rack_timeout_service_timeout_seconds: 9_999_999_999
```

Comment:
I think you want a string here. Also, we can probably make is something more reasonable, like 10 minutes

Reply:
it's valid YAML! and our config parser just calls `Integer()` on it anyways

```
require 'yaml'
YAML.load <<-STR
rack_timeout_service_timeout_seconds: 9_999_999_999
STR
=> {"rack_timeout_service_timeout_seconds"=>9999999999}
```

https://github.com/18F/identity-idp/blob/0f018a54e61b84e1f44e09d1114a33547f120390/lib/identity_config.rb#L18-L20

Reply:
Ah sweet. There used to be something that yelled at you when you put non string values in here. Maybe that went away with Figaro.

Reply:
Yeah, it was a Figaro-ism because it also had to be able to write the value to the system environment variable